oslo_cache/backends/redis.py

```diff
@@ -0,0 +1,45 @@
+from dogpile.cache.backends import redis as redis_backend
+from oslo_cache import core
+__all__ = [
+    'RedisSentinelTLSBackend'
+]
+class RedisSentinelTLSBackend(redis_backend.RedisSentinelBackend):
+    def __init__(self,arguments):
```

Comment:
```suggestion
    def __init__(self, arguments):
```

https://peps.python.org/pep-0008/#pet-peeves